docs/adr/adr_002_packaging_releasing_admin_UI.md

```diff
@@ -0,0 +1,185 @@
+3. Tag a new Kinto version (Github Action publishes the package on Pypi)
+4. Upgrade the Kinto version in Remote Settings
+5. Release and deploy Remote Settings
+
+```mermaid
+sequenceDiagram
+    actor dev
+    dev->>Kinto Admin: Tag release
+    dev->>Kinto: Update Kinto Admin VERSION file
+    dev->>Kinto: Tag release
+    Kinto->>PyPI: Build with new Kinto Admin, release to PyPI
+    Dependabot->>Remote Settings: Bump Kinto
+    dev->>Remote Settings: Tag release
+    Remote Settings->>Prod: Build with new Kinto, release to prod
+```
+
+Each step can take a certain amount of time, since they may involve pull-requests, review, and coordination.
+
+We want to reduce the amount of time and efforts that are necessary for a change in the Kinto Admin UI repository to be deployed and available to our production users.
+
+## Decision Drivers
+
+In order to choose our solution we considered the following criteria:
+
+- **Delivery Efforts**: Low → High: Amount of efforts or release steps.
+- **Complexity**: Low → High
+- **Version Consistency**: Low → High: Whether our users obtain a consistent experience between versions deployed on servers and pulled containers.
+
+## Considered Options
+
+1. [Option 0 - Do nothing](#option-0---do-nothing)
+1. [Option 1 - Overwrite Kinto Admin version in Kinto python package](#option-1---overwrite-kinto-admin-version-in-kinto-python-package)
+1. [Option 2 - Add option to override embedded Kinto Admin version](#option-2---add-option-to-override-embedded-kinto-admin-version)
+1. [Option 3 - Decouple Kinto Admin from container](#option-3---decouple-kinto-admin-from-container)
+1. [Option 4 - Merge Kinto and Kinto Admin repositories](#option-4---merge-kinto-and-kinto-admin-repositories)
+
+## Decision Outcome
+
+Chosen option: **Option 2** because it reduces the amount of steps significantly, but does not introduce any additional complexity. It is flexible and even optional.
+
+## Pros and Cons of the Options
+
+### Option 0 - Do nothing
+
+See problem statement :)
+
+**Delivery Efforts**: High. 3 release steps.
+
+**Complexity**: Low. Nothing to do.
+
+**Version Consistency**: High. Same version in Kinto and Remote Settings.
+
+
+### Option 1 - Overwrite Kinto Admin version in Kinto python package
+
+With this solution, we introduce a command in the Remote Settings repository that:
+
+1. pulls the Kinto Admin sources at a specific version ([source](https://github.com/Kinto/kinto/blob/master/scripts/build-kinto-admin.sh))
+2. build the assets (Node)
+3. overwrites the assets that are bundled with the Kinto package (ie. in `lib/python3.11/site-packages/kinto/plugins/admin/build/`) with the recent ones
+
+This way the Kinto Admin version does not have to necessarily be the same as the one bundled in the Kinto Python package.
+
+> Note: In the Remote Settings repository, the version of the Kinto Admin could be specified in a `package.json` file so that Dependabot can automatically open pull-requests to upgrade it.
```

Comment:
I think it'd be good to look into writing our own automation, maybe a Github Action like:

- chron job to check the version of https://github.com/Kinto/kinto-admin/blob/f6ba92b39645b655d9fc35a694bbf4e9ccd1bc58/package.json#L3 in the latest release
- compare that with some Kinto Admin `VERSION` file in Remote Settings
- if the version from `package.json` is greater than `VERSION`, open a new PR to update the `VERSION` file

```
git checkout -b `kinto-admin/$VERSION`

echo "$release_version" > VERSION
git add .
git commit -m "Update Kinto Admin version to $VERSION"
git push origin $BRANCH_NAME

gh pr create \
  --body "" \
  --title "Update Kinto Admin version to $VERSION" \
  --head "$BRANCH_NAME" \
  --base "main"
```

We could do the same thing in Kinto as well

Reply:
Can dependabot check the current release on github? If not I feel like writing that hook would be pretty easy to create something to do that. We'd just need to hit the latest release URL: http://github.com/Kinto/kinto-admin/

Reply:
> We don't publish Kinto Admin to NPM

Oh, I was convinced we would still do it, but that it would be done automatically on tagging in the kinto-admin repo. 

What prevents us from doing this and piggyback Dependabot pull-requests?

Reply:
> What prevents us from doing this and piggyback Dependabot pull-requests?

Since it's a project and not a component anymore, it does not make sense to publish it on npm anymore (since v2.0.0)